apps/extension/src/ui/domains/Account/AccountAdd/AccountAddLedger/AddLedgerSelectNetwork.tsx

```diff
@@ -125,6 +125,20 @@ export const AddLedgerSelectNetwork = () => {
             <p className="text-body-secondary mt-6">
               {t("Please note: a Ledger account can only be used on a single network.")}
             </p>
+            {["polkadot", "polkadot-asset-hub"].includes(chain?.id ?? "") && (
+              <p className="text-body-secondary mt-6">
+                {t(
+                  "Additionally, a given Ledger account cannot be added in Talisman for both Polakdot and Polkadot Asset Hub as they would share the same address. Adding one would remove the other, and vice versa."
```

Comment:
Copycheck:

`Adding the same Ledger account to both Polkadot and Polkadot Asset Hub results in them overriding each other.`